### PR TITLE
Issue #385 Document minimum Windows version

### PR DESCRIPTION
**Pull request by robin-owen**

### CODE REVIEW

File: docs/source/getting-started/content/proc_installing-codeready-containers.adoc [hunk 1]
```diff
@@ -14,7 +14,10 @@ These requirements must be met in order to run OpenShift in the {prod} virtual m
 Other applications running on top of OpenShift may require additional resources to run.
 ====
 
-* {prod} requires installation of the following software packages:
+* On {msw}, {prod} requires the Windows 10 Fall Creators Update (version 1709).
```
Comment (robin-owen) [inline docs/source/getting-started/content/proc_installing-codeready-containers.adoc]: Should we mention Hyper-V?

Comment (anjannath) [reply docs/source/getting-started/content/proc_installing-codeready-containers.adoc]: Its Okay not to mention about Hyper-V. @gbraad WDYT? we don't mention `kvm` module should be loaded. These are also taken care of by the preflight checks.

Comment (gbraad) [reply docs/source/getting-started/content/proc_installing-codeready-containers.adoc]: This is OK. We require Windows 10. No need to be SOOOOOOO precise to detail why, as that is part of the rest of the installation. The native target already points to this 'need'.

**Comment by cfergeau** [comment]
And we need the 64 bit edition, 32 bit is not going to be enough for hyperv.

**Pull request merged**